bentoctl_lambda/create_deployable.py

```diff
@@ -57,6 +57,13 @@ def create_deployable(
     with bento_metafile.open("r", encoding="utf-8") as metafile:
         info = BentoInfo.from_yaml_file(metafile)
 
+    enable_grpc = False
+    if hasattr(info.python, "extras_require"):
```

Comment:
this should not be the "config" for using gRPC. We should add a field to the configuration file to indicate if user want to serve a gRPC endpoint